standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/columnstats/ColStatsBuilder.java

```diff
@@ -103,6 +105,12 @@ public ColStatsBuilder<T> hll(String... values) {
     return this;
   }
 
+  public ColStatsBuilder<T> hll(double... values) {
+    HyperLogLog hll = StatisticsTestUtils.createHll(values);
+    this.bitVector = hll.serialize();
```

Comment:
This is meant to `this.kll = kll.toByteArray();`?

Reply:
No, HLL is different from KLL, it's used for counting distinct values, the method naming is different because HLL has a in-house implementation in Hive while KLL comes from Apache datasketches library